nxdrive/client/remote_client.py

```diff
@@ -95,14 +96,18 @@ def __init__(
 
         installation_type = "system-wide" if Options.system_wide else "user-only"
 
+        nx_metrics = {
+            "installation.type": installation_type,
+            "execution.profile": Options.exec_profile,
+            "execution.session.uid": str(uuid4()),
```

Comment:
I think the session ID should be created outside of the remote client. It should be set at the very beginning, maybe in `__main__.py::main()` we should set `Options.session_uid` and use that value where needed?

Reply:
Ping ⏫